frontend/src/metabase/components/form/widgets/FormToggleWidget.jsx

```diff
@@ -1,24 +1,10 @@
 import React from "react";
-import { t } from "ttag";
 
 import Toggle from "metabase/components/Toggle";
 
-const HIDDEN_STYLE = { height: 0, overflow: "hidden" };
-
-const FormToggleWidget = ({
-  field,
-  horizontal = false,
-  showEnabledLabel = !horizontal,
-}) => (
-  <div className="flex align-center">
+const FormToggleWidget = ({ field }) => (
+  <div>
```

Comment:
Do we still need this `<div>`?

Reply:
I’ll make sure it’s not doing some stealth layout and if not remove it.